boards/arm/thingy91_nrf9160/CMakeLists.txt

```diff
@@ -18,3 +18,5 @@ if(CONFIG_BOARD_THINGY91_NRF9160_NS)
 	# because it is used in other CMake files.
 	set(PM_STATIC_YML_FILE ${CMAKE_CURRENT_LIST_DIR}/pm_static.yml CACHE INTERNAL "")
 endif()
+
+zephyr_library_sources(nrf52840_reset.c)
```

Comment:
Should this file be conditionally included based on node status ?

See: https://github.com/nrfconnect/sdk-zephyr/blob/c932ad1952970987301719d927fc0232116e908f/cmake/extensions.cmake#L2604-L2619

Reply:
Done.

Reply:
Did not do this the right way.

Reply:
Should be OK now!